File: Java/bowling_game_kata/src/main/java/model/Player.java
package model;

public class Player {
    private int score;

    private boolean strike;
    private int bonusRolls;

    public Player() {
        this.score = 0;
    }

    public int getScore() {
        return score;
    }

    public void addScore(int pinsKnockedDown) {
        this.score += pinsKnockedDown;
    }

    public void setBonusRolls(int bonusRolls) {
        this.bonusRolls = bonusRolls;
    }

    public int getBonusRolls() {
        return bonusRolls;
    }

    public void addBonus(int pinsKnockedDown) {
        if(bonusRolls > 0){
            bonusRolls -= 1;
            score += pinsKnockedDown;
        }
    }
}

File: C#/Bowling_Game/Model/Player.cs
using System;

namespace Bowling_Game.Model {
    public class Player {
        private int score;

        public Player() {
            score = 0;
        }

        public int GetScore() {
            return score;
        }

        public void AddScore(int points) {
            if (points >= 0) { 
                score += points;
            }
        }
    }
}
